content/webapp/utils/requesting.ts

```diff
@@ -18,10 +19,26 @@ const locationIsRequestable = (location: PhysicalLocation): boolean => {
   );
 };
 
-export const itemIsRequestable = (item: PhysicalItem): boolean => {
+export const itemIsRequestable = (
+  item: PhysicalItem,
+  offsiteRequesting = false
+): boolean => {
   // ok because there is only one physical location in reality
   const physicalLocation = getFirstPhysicalLocation(item);
-  return !!physicalLocation && locationIsRequestable(physicalLocation);
+  // we know an item is offsite/deepstore if its 1st available date is more than 3 days in the future
+  const isOffsiteDeepstoreItem =
```

Comment:
Not really blocking and maybe unnecessary, but perhaps we can avoid making the deepstore connection by thinking about this as not making things with a long lead time requestable.

Reply:
Changed the name of the variable from `isOffsiteDeepstoreItem` to `hasLongLeadtime` and adjusted the "long" from 3 days to 9, in case we have bank holidays and the "short" library lead time ends up being more than 3 days as @rcantin-w  pointed out below.